packages/manager/modules/web-paas/src/details/user-licences/user-licences.controller.js

```diff
@@ -1,11 +1,20 @@
+import { set } from 'lodash';
 import { ADDON_TYPE } from '../../web-paas.constants';
+import { CHANGE_OFFER_IMPRESSION_TRACKING_DATA } from './user-licences.constants';
 
 export default class {
+  /* @ngInject */
+  constructor(atInternet) {
+    this.atInternet = atInternet;
+  }
+
   $onInit() {
     this.ADDON_TYPE = ADDON_TYPE;
+    this.CHANGE_OFFER_IMPRESSION_TRACKING_DATA = CHANGE_OFFER_IMPRESSION_TRACKING_DATA;
```

Comment:
Do we need to add it as a property, it seems only used in controller so we can directly use imported value